### PR TITLE
Updated gems

### PR DESCRIPTION
**Pull request by RobertAKARobin**
Kept getting rake errors.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -25,5 +25,6 @@ class Application < Rails::Application
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.autoload_paths += %W(#{config.root}/lib)
     config.active_record.raise_in_transactional_callbacks = true
+    config.web_console.development_only = false
```
Comment (jshawl) [inline config/application.rb]: wut

Comment (RobertAKARobin) [inline config/application.rb]: It kept throwing an error about that at me. Of course, now I can't reproduce it.

Comment (RobertAKARobin) [inline config/application.rb]:
```
$ RAILS_ENV=test rake db:migrate
Web Console is activated in the test environment, which is
usually a mistake. To ensure it's only activated in development
mode, move it to the development group of your Gemfile:

    gem 'web-console', group: :development

If you still want to run it the test environment (and know
what you are doing), put this in your Rails application
configuration:

    config.web_console.development_only = false
```

Comment (RobertAKARobin) [inline config/application.rb]:
```
$ rspec
Web Console is activated in the test environment, which is
usually a mistake. To ensure it's only activated in development
mode, move it to the development group of your Gemfile:

    gem 'web-console', group: :development

If you still want to run it the test environment (and know
what you are doing), put this in your Rails application
configuration:

    config.web_console.development_only = false
```

Comment (andrewsunglaekim) [inline config/application.rb]: i think adam fixed this in https://github.com/ga-dc/garnet/pull/546

Comment (RobertAKARobin) [inline config/application.rb]: Hah, so he did. I'll close this in favor of that.

**Pull request closed without merging**